Podfile.lock

```diff
@@ -40,6 +40,6 @@ SPEC CHECKSUMS:
   TwilioVoice: 098a959181d4607921f5822d3c9f13043ea4075b
   WebRTC-lib: 508fe02efa0c1a3a8867082a77d24c9be5d29aeb
 
-PODFILE CHECKSUM: 40bea65dec1e511d94b8bce63fe030eae8e04f7b
+PODFILE CHECKSUM: a7d8277ba69e9b60beed850d1c25b39b6b0a759b
 
-COCOAPODS: 1.15.0
+COCOAPODS: 1.12.1
```

Comment:
I think this Podfile.lock should be either not sent for review or local cocoapods is updated.

Reply:
I would update to the latest version. Now it's `1.15.2`

Reply:
fixed [530d2a7](https://github.com/salemove/ios-sdk-widgets/pull/887/commits/530d2a730146ae74e970d12b893aaef1b9707f5f)